lib/core/exceptions.toit

```diff
@@ -169,14 +169,13 @@ program_failure_ bci:
 
 /**
 Signals an 'as' failure.
-$id might be either:
-- a class name of type $string
-- an absolute BCI of the 'as' check.
+The absolute bytecode index of the 'as' check is
```

Comment:
Same here: when there is no instantiation of the class, then we call the `as_check_failure_` method directly with the name of the class (since there isn't any class-id to check against).